<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bigglesworth Manual</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="contentWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="homeBtn">
           <property name="icon">
            <iconset theme="go-home">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset theme="arrow-left">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fwdBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset theme="arrow-right">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="HelpBrowser" name="helpBrowser">
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpBrowser</class>
   <extends>QTextBrowser</extends>
   <header>bigglesworth.help</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
